src/bitcoin-cli.cpp

```diff
@@ -610,10 +627,14 @@ class NetinfoRequestHandler : public BaseRequestHandler
         "  send     Time since last message sent to the peer, in seconds\n"
         "  recv     Time since last message received from the peer, in seconds\n"
         "  txn      Time since last novel transaction received from the peer and accepted into our mempool, in minutes\n"
+        "           \".\" - the peer requested we not relay transactions to it (relaytxes is false)\n"
```

Comment:
**Minor nit**: the `hb` column currently uses the '`.`' and '`*`' convention to indicate details regarding our preference and the remote peer's respectively. The processed addresses column (`addrp`) also maintains this convention. I would suggest `txn` keep that consistency by using '`*`' instead of '`.`' to indicate the remote peer's request regarding us relaying `txs` to them.

@jonatack what do you think?

Reply:
Good point. I have been asking myself the same but hesitated because the `*` seemed a bit "loud" in the `txn` column. I agree that it's better that the logic be coherent. Done, thanks for the thoughtful feedback!